CHANGELOG.md

```diff
@@ -4,23 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.9.0
+### Added
+* CLI/Service: Added [Service Managed Job Templates](docs/declarative-templates.md) as a preview feature.  (enable via `onefuzz config --enable_feature job_templates`).  [#226](https://github.com/microsoft/onefuzz/pull/296)
+* Service/agent: Added support for unmanaged nodes.  This paves the way for _bring your own compute_ for fuzzing.  [#318](https://github.com/microsoft/onefuzz/pull/318)
```

Comment:
Can you elaborate on the level of support, stability? Are we calling this a preview feature?

Reply:
It's functional and stable, though not exposed in a meaningful fashion.  I'm fine with it as is, as it will get flushed out significantly more in the near future.

Reply:
Oh sorry, I meant elaborate in the changelog. Like "Added **internal**/**preview** support for [...]".